pkg/api/v1beta1/types.go

```diff
@@ -445,8 +446,10 @@ const (
 
 // TODO: add LastTransitionTime, Reason, Message to match NodeCondition api.
 type PodCondition struct {
-	Kind   PodConditionKind `json:"kind" description:"kind of the condition, currently only Ready"`
-	Status ConditionStatus  `json:"status" description:"status of the condition, one of Full, None, Unknown"`
+	// Kind is the kind of the condition
```

Comment:
Ok, though I don't think we're very consistent about it, and it just creates 2 things to update and keep in sync.